features/step_definitions/controllers-steps.js

```diff
@@ -180,6 +180,10 @@ Then('The response should contains a {string} equals to undefined', async functi
   should(this.props.response[key]).equal(undefined);
 });
 
+Then('The property {string} of the response should be equal to: {string}', async function (key, value) {
```

Comment:
This step could be adapted to be more generic like [the one matching the result](https://github.com/kuzzleio/kuzzle/blob/master/features/step_definitions/controllers-steps.js#L87)